### PR TITLE
Just a few typo corrections:)

### PR DESCRIPTION
**Pull request by jessp01**
These little corrections aside, just wanted to convey how much I enjoyed reading this one:) Well done, Larry and thank you for your dedicated work over the years.

### CODE REVIEW

File: content/legacy/_pub_2007_12_06_soto-11.md [hunk 1]
```diff
@@ -57,17 +57,17 @@ So I'm afraid all I can do is give you my own worm's eye view of the past, the p
...
 
-But I'm not going to apologize for that. All language designers have their occasional idiosyncracies. I'm just better at it than most. `:-)`
+But I'm not going to apologize for that. All language designers have their occasional idiosyncrasies. I'm just better at it than most. `:-)`
 
 ### RSTS BASIC/PLUS
 
-Anyway, when I was a RSTS programmer on a PDP-11, I certainly treated BASIC as a scripting language, at least in terms of rapid prototyping and process control. I'm sure it warped my brain forever. Perl's statement modifiers are straight out of BASIC/PLUS. It even had some cute sigils on the ends of its variables to distinguish string and integer from floating point.
+Anyway, when I was a `RSTS` programmer on a PDP-11, I certainly treated BASIC as a scripting language, at least in terms of rapid prototyping and process control. I'm sure it warped my brain forever. Perl's statement modifiers are straight out of BASIC/PLUS. It even had some cute sigils on the ends of its variables to distinguish string and integer from floating point.
```
Comment (briandfoy) [inline content/legacy/_pub_2007_12_06_soto-11.md:64]: Undo the quoting here. Plain text is fine.

File: content/legacy/_pub_2007_12_06_soto-11.md [hunk 2]
```diff
@@ -57,17 +57,17 @@ So I'm afraid all I can do is give you my own worm's eye view of the past, the p
...
 
 ### Unix?
 
-I remember one day our computer center got a letter from Bell Labs telling us that we could get a tape of Unix V6 for cheap, only $100 because they were coming out shortly with V7. We all looked at each other and said, Why would we ever want to use this thing called Unix? We have RSTS.
+I remember one day our computer center got a letter from Bell Labs telling us that we could get a tape of Unix V6 for cheap, only $100 because they were coming out shortly with V7. We all looked at each other and said, Why would we ever want to use this thing called Unix? We have `RSTS`.
```
Comment (briandfoy) [inline content/legacy/_pub_2007_12_06_soto-11.md:70]: Undo the quoting here.

File: content/legacy/_pub_2007_12_06_soto-11.md [hunk 3]
```diff
@@ -89,7 +89,7 @@ Once I got into industry, I wrote a compiler in Pascal for a discrete event simu
 
 ### Unix, shell
 
-After several years I finally became acquainted with Unix and its various scripting languages. OK, to be more precise, BSD, and csh.
+After several years I finally became acquainted with Unix and its various scripting languages. OK, to be more precise, `BSD`, and `csh`.
```
Comment (briandfoy) [inline content/legacy/_pub_2007_12_06_soto-11.md:92]: Undo the quoting here.

**Review by briandfoy — CHANGES_REQUESTED** [review]
I'll take the typo fixes, but I'd prefer to leave the formatting as is.

File: content/legacy/_pub_2007_12_06_soto-11.md [hunk 4]
```diff
@@ -318,4 +318,4 @@ In my completely unbiased opinion, that would be Perl 6. `:-)`
...
 
-Among the generalists, the conventional wisdom is that the worse-is-better approach is more adaptive. Personally, I get a little tired of the argument: My worse-is-better is better than your worse-is-better because I'm better at being worser! Is it really true that the worse-is-better approach always wins? With Perl 6 we're trying to sneak one better-is-better cycle in there and hope to come out ahead before reverting to the tried and true worse-is-better approach. Whether that works, only time will tell.
+Among the generalists, the conventional wisdom is that the worse-is-better approach is more adaptive. Personally, I get a little tired of the argument: My worse-is-better is better than your worse-is-better because I'm better at being worse! Is it really true that the worse-is-better approach always wins? With Perl 6 we're trying to sneak one better-is-better cycle in there and hope to come out ahead before reverting to the tried and true worse-is-better approach. Whether that works, only time will tell.
```
[the remark this answers is not in the document]
Comment (jessp01) [reply content/legacy/_pub_2007_12_06_soto-11.md]: I believe worser is not actually a word:) Oxford does not list it and `aspell` doesn't recognise it. Nonetheless, I've done as you've asked.

Cheers,

**Comment by briandfoy** [comment]
I ended up pulling these changes into 6144387921dd4d9558c6dc0730034a56f752e455 so it would be one commit instead of a commit to undo things.

Thanks for the work,

File: content/legacy/_pub_2007_12_06_soto-11.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (briandfoy) [reply content/legacy/_pub_2007_12_06_soto-11.md]: Larry often uses words that aren't words.

**Pull request closed without merging**